### PR TITLE
Documentation/readme

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies

Description taken from #109

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,12 @@ using: `kustomize build --enable_alpha_plugins overlays/test | conftest test --p
 
 ### pushing a release to production
 
-### developing a new component
+### Developing a new component
+
+#### Component secrets
+We use [sops](https://github.com/mozilla/sops#sops-secrets-operations) for creating secrets with encryption.
+Contributor first needs to have the secret in the manifest file to run `sops` on it for encryption
+(because we cannot have that secret file mistakenly pushed to the repo).
```
Comment (harshad16) [inline README.md:34]:
```suggestion
We use [sops](https://github.com/mozilla/sops#sops-secrets-operations) for encrypting secrets.
For encrypting the secrets, the contributor can do the following:
 - first, create a `secrets.yaml` at the root level. (git would ignore the file)
 - Then run sops command to encrypt them to the desired application subdirectory.
  Example: `sops -e secrets.yaml > core/overlays/test/secrets.enc.yaml`
 - Make sure contribute the encrypted **secrets.enc.yaml** .
```
How about this?

File: README.md [hunk 2]
```diff
@@ -26,7 +26,15 @@ using: `kustomize build --enable_alpha_plugins overlays/test | conftest test --p
...
 
-### developing a new component
+### Developing a new component
+
+#### Component secrets
+We use [sops](https://github.com/mozilla/sops#sops-secrets-operations) for encrypting secrets.
+For encrypting the secrets, the contributor can do the following:
+ - First, create a `secrets.yaml` at the root level. (git would ignore the file)
+ - Then run sops command to encrypt them to the desired application subdirectory.
+  Example: `sops -e secrets.yaml > core/overlays/test/secrets.enc.yaml`
```
Comment (harshad16) [inline README.md:36]:
```suggestion
  Example: `sops -e secrets.yaml > {component}/overlays/test/secrets.enc.yaml`
```

Comment (harshad16) [reply README.md:36]: > why do component secrets go into the core directory?

it was an example, however following suggestion can be used.

**Comment by xtuchyna** [comment]
heya @goern is it alright now?

**Pull request merged**